packages/cli/src/commands/optimize/optimize.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @flow
+ */
+
+const fs = require('fs').promises;
```

Comment:
Please use regular `fs`, `fs.promises` is experimental API and only available in Node 10+, while we target Node 8.3+

Reply:
not using promises